### PR TITLE
Google maps feature added to chatbot successfully.

### PR DESCRIPTION
**Pull request by SinghHimadri**
## Issue that this pull request solves
#330 
Closes: # (issue number)

## Proposed changes

Brief description of what is fixed or changed
User needs to type /maps {origin},{destionation},{travel mode} and a link will be replied by the bot by which clicking on it will redirect the user to google maps auto filled source dest and travel mode.If google maps is not installed then it will open in browser.
## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
Screenshot from 2020-05-19 00-00-19

WhatsApp Image 2020-05-19 at 12 00 45 AM (1)

After clicking on link
WhatsApp Image 2020-05-19 at 12 00 45 AM

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: wplay/chatbot.py [hunk 1]
```diff
@@ -35,6 +35,17 @@ async def Bot(last_Message):
             print("Taking screenshot of google image for query")
             return "Sending you screenshot"
 
+        elif command_arg[0] == "maps":
+            query = "".join(command_arg[1])
+            map_parameters_list = query.replace(" ", "")
```
Comment (rpotter12) [inline wplay/chatbot.py:40]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5545098)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- wplay/chatbot.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5545098&bid=18178984)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- wplay/chatbot.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5545098&bid=18178984)

**Comment by SinghHimadri** [comment]
@rpotter12 Please review my PR.

**Pull request merged**